byon/files/seaclouds-catalog.bom

```diff
@@ -186,13 +186,14 @@ brooklyn.catalog:
       type: org.apache.brooklyn.entity.brooklynnode.BrooklynNode
       id: deployer
       name: Deployer
-      version: 0.8.0-incubating
+      version: 0.9.0-SNAPSHOT
 
       brooklyn.config:
         onExistingProperties: do_not_use
         brooklynnode.webconsole.nosecurity: true
         brooklynnode.classpath:
-          - https://s3-eu-west-1.amazonaws.com/seaclouds-deployer/deployer-0.8.0-20151007.193920-32.jar
+          - https://s3-eu-west-1.amazonaws.com/seaclouds-deployer/deployer-0.8.0-20150812.15.43.24-32-b090.jar
+          - https://s3-eu-west-1.amazonaws.com/tosca-temporal-integration/brooklyn-tosca-transformer-0.9.0-SNAPSHOT.jar
```

Comment:
Really, this jar contains brooklyn-tosca/brooklyn-tosca-transformer logic. So, I could rename this artifact